packages/victory-legend/src/index.d.ts

```diff
@@ -44,7 +50,6 @@ export interface VictoryLegendProps
   events?: EventPropTypeInterface<"data" | "labels" | "parent", StringOrNumberOrCallback>[];
   gutter?: number | { left: number; right: number };
   itemsPerRow?: number;
-  labelComponent?: React.ReactElement;
   orientation?: "horizontal" | "vertical";
   rowGutter?: number | Omit<BlockProps, "left" | "right">;
   style?: VictoryStyleInterface & { title?: VictoryStyleObject };
```

Comment:
`VictoryLegend` also has a `border` style

Reply:
Ah, this is included in https://github.com/FormidableLabs/victory/blob/22af3f4c17a17bd3734ba31fc2bed4ad252bbcfe/packages/victory-core/src/index.d.ts#L133